### PR TITLE
add travis cl configuation

### PR DESCRIPTION
**Pull request by JavierJia**
@chenlica set up the Travis CI file.

### CODE REVIEW

**Comment by chenlica** [comment]
@JavierJia : I saw "All checks have failed." Do you know why?  Thank you.

**Comment by JavierJia** [comment]
@chenlica Fixed.

**Comment by chenlica** [comment]
Thanks, @JavierJia .  Let's meet in person how to maintain this service and its related account info.

File: CMakeLists.txt [hunk 1]
```diff
@@ -71,7 +71,7 @@ SET(BUILD_SERVER ON CACHE BOOL
     "Build the Srch2Server executable. Defaults to ON")
 SET(BUILD_RELEASE ON CACHE BOOL "Build Release. Default ON")
 SET(BUILD_TEST ON CACHE BOOL "Build tests. Default ON")
-SET(BUILD_UTILITY OFF CACHE BOOL "Build utilities. Default OFF")
+SET(BUILD_UTILITY ON CACHE BOOL "Build utilities. Default ON")
```
Comment (chenlica) [inline CMakeLists.txt:74]: @surendra-bisht : please check if this change is related to the discussion we had with Saurav.

Comment (chenlica) [inline CMakeLists.txt:74]: Comment by @JavierJia : we turn the utility build "ON" in order to build a Chinese dictionary that is needed by the analyzer test.

**Comment by chenlica** [comment]
+1

**Pull request merged**